sdk/core/azure-core/test/ut/request_id_policy.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/http/policies/policy.hpp>
+#include <azure/core/internal/http/pipeline.hpp>
+
+#include <gtest/gtest.h>
+
+using namespace Azure::Core;
+using namespace Azure::Core::Http;
+using namespace Azure::Core::Http::Policies;
+using namespace Azure::Core::Http::Policies::_internal;
+
+namespace {
+class NoOpPolicy : public HttpPolicy {
+public:
+  std::unique_ptr<HttpPolicy> Clone() const override { return std::make_unique<NoOpPolicy>(*this); }
+
+  std::unique_ptr<RawResponse> Send(Request&, NextHttpPolicy, Azure::Core::Context const&)
+      const override
+  {
+    return nullptr;
+  }
+};
+
+constexpr char const* RequestIdHeaderName = "x-ms-client-request-id";
```

Comment:
static?

Reply:
Does not really matter for constexpr, but why not, I'll add it.

Reply:
In this case, variables not inside a class or function , but inside a namespace should not be static.